### ISSUE TITLE
Sentinel's luminance affects transparency on settings that aren't Fabulous

### ISSUE THREAD

**Issue by GameGuruGreenGryphon** [gamma-delta/HexMod#63]
### Versions
Minecraft 1.18.2
Forge: 40.0.34 and 40.0.45
Hex: **minor-patterns** branch commit [b89d2b3](https://github.com/gamma-delta/HexMod/commit/b89d2b36b4023440b478c5f08b34bee01b2d33fb) "maybe fix particles rendering through walls"
Paucal: **main** branch commit [92990a8](https://github.com/gamma-delta/PAUCAL/commit/92990a8f19baa918de6a0b53f4f9d631a13b1905) "[Release] Make headpatting less janky"
### Issue
![image](https://user-images.githubusercontent.com/87587913/162817597-e31e3fb3-9ee2-4eae-a93f-d32be3fe3a5a.png)
This is the black pigment that I'm showing off in this screenshot.
A colorizer's luminance is directly tied to how transparent the colors look on fast and fancy graphics settings, rather than how bright the color actually is.
This affects every single colorizer in the game that isn't fully luminant. Colorizers that would normally be pretty are slightly to very transparent and lacking their color.

**Comment by yrsegal** [comment]
this is not true for all computers, mind

**Comment by gamma-delta** [comment]
yeah given this is a zillion years old and it's weird rendering BS i'm going to close it, mostly cause i have no idea how one would fix it and we've changed that code since.

**Comment by yrsegal** [comment]
This was half-fixed by https://github.com/gamma-delta/HexMod/commit/8cbcea28334eeff06d0c1553d51a2e0b7ccfe78e

I have NO idea how to fix the other half. I'm gonna see if willie does
